#!/usr/bin/env python
PACKAGE = "hanse_ballfollowing"
import roslib;roslib.load_manifest(PACKAGE)

from math import pi

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("minSize", double_t, 0, "", 0.05, 0.0, 1.0)
gen.add("maxSize", double_t, 0, "", 0.4, 0.0, 1.0)
gen.add("fwSpeed", double_t, 0, "", 0.5, 0.0, 1.0)
gen.add("kpAngle", double_t, 0, "", 0.2, -1.0, 1.0)
gen.add("lostFactor", double_t, 0, "", 3, 0, 10)


exit(gen.generate(PACKAGE, "ballfollowing", "BallFollowing"))
